docs/security/seccomp.md

```diff
@@ -81,3 +80,84 @@ profile.
 $ docker run --rm -it --security-opt seccomp:unconfined debian:jessie \
     unshare --map-root-user --user sh -c whoami
 ```
+
+### Why we made the default profile block certain syscalls
+
+Below is a list of certain syscalls and why we are blocking them in the default
+seccomp profile.
```

Comment:
Given that we go for a whitelist now, we may need to mention that this list is a non exhaustive list?